.github/workflows/deploy.yml

```diff
@@ -0,0 +1,47 @@
+# run deploy only when pushing new tags
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: Deploy
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+
+      - name: Gradle cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-jdk8-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-jdk8
+            
+      - name: Create Artifacts
+        run: ./gradlew assemble
+        # just to ensure it compiles; if it doesn't then don't continue
+
+      - name: Configure GPG Key Properties
```

Comment:
Not that is really matters, but should this be PGP instead of GPG?

Reply:
Originally it was using GPG (GnuPG) to wrangle the key but that got too complicated. I can remove the term altogether

Reply:
Ah gotcha, I saw `useInMemoryPgpKeys` and had only heard of PGP keys not GPG keys so I just wanted to make sure